File: demo/src/main/java/com/example/Direccion.java
package com.example;

public class Direccion {
    private String calle, nomenclatura, barrio, ciudad, edificio, apto;

    public Direccion() {}
    public Direccion(String calle, String nomenclatura, String barrio, String ciudad, String edificio, String apto) {
        this.calle = calle;
        this.nomenclatura = nomenclatura;
        this.barrio = barrio;
        this.ciudad = ciudad;
        this.edificio = edificio;
        this.apto = apto;
    }

    public void setCalle(String c) { calle = c; }
    public void setNomenclatura(String n) { nomenclatura = n; }
    public void setBarrio(String b) { barrio = b; }
    public void setCiudad(String ci) { ciudad = ci; }
    public void setEdificio(String e) { edificio = e; }
    public void setApto(String a) { apto = a; }

    public String getCalle() { return calle; }
    public String getNomenclatura() { return nomenclatura; }
    public String getBarrio() { return barrio; }
    public String getCiudad() { return ciudad; }
    public String getEdificio() { return edificio; }
    public String getApto() { return apto; }

    public String toString() {
        return calle + " " + nomenclatura + ", " + barrio + ", " + ciudad + ", Ed. " + edificio + ", Apt. " + apto;
    }
}

File: python/Direccion.py
class Direccion:
    def __init__(self, calle="", nomenclatura="", barrio="", ciudad="", edificio="", apto=""):
        self.calle = calle
        self.nomenclatura = nomenclatura
        self.barrio = barrio
        self.ciudad = ciudad
        self.edificio = edificio
        self.apto = apto

    def set_calle(self, calle): self.calle = calle
    def set_nomenclatura(self, nomenclatura): self.nomenclatura = nomenclatura
    def set_barrio(self, barrio): self.barrio = barrio
    def set_ciudad(self, ciudad): self.ciudad = ciudad
    def set_edificio(self, edificio): self.edificio = edificio
    def set_apto(self, apto): self.apto = apto

    def get_calle(self): return self.calle
    def get_nomenclatura(self): return self.nomenclatura
    def get_barrio(self): return self.barrio
    def get_ciudad(self): return self.ciudad
    def get_edificio(self): return self.edificio
    def get_apto(self): return self.apto

    def __str__(self):
        return f"{self.calle} {self.nomenclatura}, {self.barrio}, {self.ciudad}, Ed. {self.edificio}, Apt. {self.apto}"
